src/ui/rcb-ui-inner.svelte

```diff
@@ -36,6 +36,14 @@
   export let purchaseOperationHelper: PurchaseOperationHelper;
   export let isInElement: boolean = false;
 
+  const shouldShowDetailsButton =
```

Comment:
point for refactor, don't like that this logic is coupled here

Reply:
Maybe add a comment for the potential refactoring?
Exposing this as a parameter of the component might be useful when we'll expose these components into `purchases-ui-js`

Reply:
done